src/main/java/bio/terra/service/resourcemanagement/azure/AzureAuthService.java

```diff
@@ -21,32 +23,59 @@ public AzureAuthService(AzureResourceConfiguration configuration) {
   }
 
   /**
-   * Return an authenticated DataLake client using key-based authentication
+   * Return an authenticated {@link DataLakeServiceClient} using key-based authentication
    *
    * @param profileModel The object containing user tenant information
-   * @param storageAccountResource The storage account that DataLake client should built from
+   * @param storageAccountResource The storage account that DataLake client should be built from
    * @return an authenticated DataLake client
    */
   public DataLakeServiceClient getDataLakeClient(
```

Comment:
It looks like this is no longer used. Is that expected?

Reply:
Yeah.  We're going to need this one for parquet files I think (since that will need DataLake type Sas tokens - there are different types)